src/actions/index.js

```diff
@@ -46,25 +45,28 @@ const ALERT_LEVEL_ERROR = 'error'
 export const downloadFileMissing = () => ({ type: DOWNLOAD_FILE_E_MISSING, alert: { message: 'error.download_file.missing', level: ALERT_LEVEL_ERROR } })
 export const downloadFileOffline = () => ({ type: DOWNLOAD_FILE_E_OFFLINE, alert: { message: 'error.download_file.offline', level: ALERT_LEVEL_ERROR } })
 
-export const openFolder = (folderId, context = FILES_CONTEXT) => {
+export const locationChange = (virtualRoot, folderId) => {
```

Comment:
Yeah, good remark: I dispatch something to alert redux global state to be updated. What is this *something*?
If I refer to the *flux* architecture, I can see that an action is something that dispatch an event to trigger a state change.

![](https://facebook.github.io/flux/img/flux-simple-f8-diagram-with-client-action-1300w.png)